src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -241,7 +230,7 @@ public JsonApiException toApiException(Throwable cause, String format, Object...
   }
 
   private String getErrorMessage(String format, Object... args) {
-    if (extendError) {
+    if (ExtendError.enabled()) {
```

Comment:
First access to `getErrorMessage()` of any `ErrorCode` instance will load `ExtendError` helper class, which will get configuration settings. This is properly synchronized as guaranteed by JVM classloading, and all further access is via properly initialized singleton.